pkg/occlient/occlient.go

```diff
@@ -892,6 +899,21 @@ func (c *Client) GetClusterServiceClasses() ([]scv1beta1.ClusterServiceClass, er
 	return classList.Items, nil
 }
 
+func (c *Client) CreateServiceInstance(componentName string, componentType string, labels map[string]string, applicationName string, projectName string) error {
```

Comment:
we are in `occlient` package it should have `componentName` or `componentType` labels